python-checks/src/test/resources/checks/stringFormatCorrectness.py

```diff
@@ -1,5 +1,5 @@
 # Due to parsing errors with formatted expressions, the rule is disabled for f-strings: see SONARPY-726
```

Comment:
I know this is not the scope of this PR, but could you take the opportunity to remove this comment?
It's been a while this is not relevant anymore.